### DISCUSSION TITLE
Home assitant Troppo Difficile per me..

### DISCUSSION THREAD

**Discussion by Miksca** [indomus/forum#3280] [HUB: Altro (Homey, iHost, SmartThings, Fibaro ecc.)]
Ciao ci ho provato e perso parecchio tempo ma per me è troppo difficile...
Io vorrei un HUB dove mettere alcune telecamere e sensori (allagamento e fumo) senza automazioni domotiche.
Ho provato HA ma è troppo complesso, adesso sono indeciso tra:
Home Kit + HomeBridge
Homey Pro
Oppure Aqara Hub M3 che mi sembra la scelta più semplice... 
Consigli? Grazie

**Comment by KC-inDomus** [comment]
Se hai difficoltà, usa Homey Pro piuttosto
